.travis.yml

```diff
@@ -37,3 +37,4 @@ matrix:
 script:
   - docker build --build-arg LLVM_VERSION=$LLVM_VERSION -t bpftrace-builder-$BASE-llvm-$LLVM_VERSION -f docker/Dockerfile.$BASE docker/
   - sudo docker run --privileged --rm -it -v $(pwd):$(pwd) -v /sys/kernel/debug:/sys/kernel/debug:rw -e STATIC_LINKING=$STATIC_LINKING -e TEST_ARGS=$TEST_ARGS bpftrace-builder-$BASE-llvm-$LLVM_VERSION $(pwd)/build-$TYPE-$BASE $TYPE -j`getconf _NPROCESSORS_ONLN`
+  - ./clang-format.sh && if [ $(git diff HEAD | wc -l) == "0" ]; then exit 0; else exit 1; fi
```

Comment:
Personal preference of my is to have the CI also tell what you did wrong, not only that it did it wrong. So including the `git diff` output as well, maybe limited to the first 500 or so lines for the big change sets.

Reply:
I've seen other projects have multiple github checks. One is the usual build stuff. The other is "clang-format". If "clang-format" is red, it means you didn't format your stuff. Would that make sense here?

Reply:
Yeah that sounds good to me